spec/api/v1/api_crops_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Api::V1::CropsController do
+
+  before(:each) do
+    @crop = FactoryGirl.create(:tomato)
+    @crop.save
+  end
+
+  describe "GET crops index" do
+    it 'fetches the crops index' do
+      get '/api/v1/crops'
+      response.should be_success
+      response.content_type.should eq("application/json")
+    end
```

Comment:
Might be a good idea to check the length of the JSON content as well, to make sure that only the specified content is contained within. That way, if we add or remove anything later, the test will fail reminding us to update it.